packages/playwright-crawler/src/internals/utils/playwright-utils.ts

```diff
@@ -19,15 +19,98 @@
  * @module playwrightUtils
  */
 
+import { readFile } from 'node:fs/promises';
 import ow from 'ow';
 import { Page, Response, Route } from 'playwright';
+import { LruCache } from '@apify/datastructures';
 import log_ from '@apify/log';
 import { validators, Request } from '@crawlee/core';
 import { Dictionary } from '@crawlee/utils';
 import { PlaywrightCrawlingContext } from '../playwright-crawler';
 
 const log = log_.child({ prefix: 'Playwright Utils' });
 
+const jqueryPath = require.resolve('jquery');
+
+const MAX_INJECT_FILE_CACHE_SIZE = 10;
+
+/**
+ * Cache contents of previously injected files to limit file system access.
+ */
+const injectedFilesCache = new LruCache({ maxLength: MAX_INJECT_FILE_CACHE_SIZE });
+
+/**
+ * Injects a JavaScript file into a Playright page.
+ * Unlike Playwright's `addScriptTag` function, this function works on pages
+ * with arbitrary Cross-Origin Resource Sharing (CORS) policies.
+ *
+ * File contents are cached for up to 10 files to limit file system access.
+ *
+ * @param page Playwright [`Page`](https://playwright.dev/docs/api/class-page) object.
+ * @param filePath File path
+ * @param {object} [options]
+ * @param {boolean} [options.surviveNavigations]
+ *   Enables the injected script to survive page navigations and reloads without need to be re-injected manually.
+ *   This does not mean, however, that internal state will be preserved. Just that it will be automatically
+ *   re-injected on each navigation before any other scripts get the chance to execute.
+ * @return {Promise<*>}
+ * @memberOf playwright
```

Comment:
please adjust the jsdoc
- remove `memberOf` tag
- move return type to the func declaration
- remove types from jsdoc that are explicit in the signature
- type the options parameter properly and move the jsdoc describing each option inside the interface 

same for `injectJQuery`